website/docs/quickstart.md

```diff
@@ -61,17 +61,22 @@ If you lose your environment token, just repeat this process to generate a new o
 
 :::warning Danger, Will Robinson
 
-If are you coming from Suborbital Compute `v0.3.3` or earlier and would like locally develop on SE2 `v0.4.0` or greater, you must first upgrade to Subo `v0.6.0` and update the deployment templates with `subo se2 deploy --reset`.
+If you're coming from Suborbital Compute `v0.3.3` or earlier and would like to locally develop on SE2 `v0.4.0` or greater, you must first [upgrade to Subo `v0.6.0`](./subo#upgrade-subo) and update the deployment templates with `subo se2 deploy --reset`.
 
 An SE2 migration tool for production deployments of Compute will be available soon.
 
 :::
 
-Next, use Subo to start your local SE2 instance. Make sure to do this within the same directory you created above! You can verify that you're in the correct directory by verifying that it contains the `docker-compose.yaml` file.
+Next, use Subo to start your local SE2 instance. Make sure to do this within the same directory you created above!
 
-```bash
-subo se2 deploy --local
-```
+- Make sure Docker is running
+- Run:
+
+  ```bash
+    subo se2 deploy --local
+    ```
```

Comment:
Should these two lines be de-dented by two spaces?

Or just inline it?

```
- Run: `subo se2 deploy --local`
```

Reply:
Oh, maybe I see what you mean: line 75 is indented by two spaces, and then the lines below are indented twice, as auto-formatted by VSCode and maybe Prettier. I'll de-indent lines 76 and 77.